### PR TITLE
Add ollama

### PR DESCRIPTION
**Pull request by boxabirds**
Very popular local LLM inference server.

### CODE REVIEW

**Comment by boxabirds** [comment]
Right so the deeplink doesn’t work but the other fields should be ok.

**Comment by osanseviero** [comment]
The only goal of this button is to work with deeplinks. The button serves no other purpose

**Review by Vaibhavs10 — COMMENTED** [review]
Hi @boxabirds - Just a heads-up we are thinking about this a bit more in our current sprint. We might have some exciting updates soon! 🤗

**Comment by julien-c** [comment]
i thinik we could revive this PR but switch to a copy-pasteable command (like llama.cpp) rather than a deeplink

**Comment by julien-c** [comment]
@boxabirds ok for you if @mishig25 from our team takes over your PR and completes it?

File: packages/tasks/src/local-apps.ts [hunk 1]
```diff
@@ -389,6 +399,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
 		deeplink: (model) => new URL(`https://models.invoke.ai/huggingface/${model.id}`),
 	},
+	ollama: {
+		prettyLabel: "Ollama",
+		docsUrl: "https://ollama.com",
```
Comment (ngxson) [inline packages/tasks/src/local-apps.ts:404]: Probably have dedicated docs on the hub for this, but we can make a follow-up PR later on

**Review by ngxson — APPROVED** [review]

File: packages/tasks/src/local-apps.ts [hunk 2]
```diff
@@ -143,6 +144,15 @@ const snippetNodeLlamaCppCli = (model: ModelData, filepath?: string): LocalAppSn
 	];
 };
 
+const snippetOllama = (model: ModelData, filepath?: string): string => {
+	if (filepath) {
+		const quantLabel = parseGGUFQuantLabel(filepath);
+		const ollamatag = quantLabel ? `:${quantLabel}` : "";
+		return `ollama run hf.co/${model.id}${ollamatag}`;
+	}
+	return `ollama run hf.co/${model.id}{{OLLAMA_TAG}}`;
```
Comment (julien-c) [inline packages/tasks/src/local-apps.ts:153]: maybe document it line 55 (where you document `GGUF_FILE`) (can be in a later PR if you prefer)

**Review by julien-c — APPROVED** [review]

File: packages/tasks/src/local-apps.ts [hunk 3]
```diff
@@ -389,6 +399,13 @@ export const LOCAL_APPS = {
 		displayOnModelPage: (model) => model.library_name === "diffusers" && model.pipeline_tag === "text-to-image",
 		deeplink: (model) => new URL(`https://models.invoke.ai/huggingface/${model.id}`),
 	},
+	ollama: {
+		prettyLabel: "Ollama",
+		docsUrl: "https://ollama.com",
+		mainTask: "text-generation",
+		displayOnModelPage: (model) => model.tags.includes("conversational") && isLlamaCppGgufModel(model),
```
Comment (julien-c) [inline packages/tasks/src/local-apps.ts:406]: rm conversational no?

Comment (mishig25) [reply packages/tasks/src/local-apps.ts]: handled in cbf44d6165284d14d74a4bb644278032176967ef as ollama also supports embedding & next-token prediction models (besides the conversational ones)

File: packages/tasks/src/local-apps.ts (hunk 2 above)
Comment (mishig25) [reply packages/tasks/src/local-apps.ts:154]: handled in c2691b00d212a2f5b811ab9136fc040724d63043

**Comment by mishig25** [comment]
@coyotte508 could you help me with correctly importing hfjs/gguf inside hfjs/tasks? 
I did https://github.com/huggingface/huggingface.js/pull/687/commits/acae2361b16331952db23ed6134904dd10b87aa1 & https://github.com/huggingface/huggingface.js/pull/687/commits/b34e76b6d36bb1af9dfafcbea128110298972a59 with command 
```
cd packages/tasks
pnpm add @huggingface/gguf@workspace:^
```

However, the tests are failing & `huggingface/gguf` is not visible inside `huggingface/tasks`

**Comment by julien-c** [comment]
> (but not thrilled to add a dep to `@huggingface/tasks`)

im guessing we could merge tasks and gguf...

**Pull request merged**